src/dolphin/workflows/config/_displacement.py

```diff
@@ -142,6 +169,24 @@ def _check_slc_files_exist(self) -> "DisplacementWorkflow":
         self.cslc_file_list = [
             Path(f) for f in sort_files_by_date(file_list, file_date_fmt=date_fmt)[0]
         ]
+
+        # # Check for tropospheric weather-model files
+        # tropo_file_list = self.troposphere_files
+        # correction_options = self.correction_options
+        # tropo_date_fmt = correction_options.tropo_date_fmt
+        # # Filter out files that don't have dates in the filename
+        # files_matching_date = [Path(f) for f in tropo_file_list if get_dates(f, fmt=tropo_date_fmt)]
+        # if len(files_matching_date) < len(tropo_file_list):
+        #     raise (
+        #         f"Found {len(files_matching_date)} weather-model files with dates like {tropo_date_fmt} in"
+        #         f" the filename out of {len(tropo_file_list)} files."
+        #     )
+
+        # # Coerce the file_list to a sorted list of Path objects
+        # self.troposphere_files = [
+        #     Path(f) for f in sort_files_by_date(tropo_file_list, file_date_fmt=tropo_date_fmt)[0]
+        # ]
```

Comment:
```suggestion
```
we can always check the git history for this if we want to come back and add it later